### PR TITLE
OCPBUGS-8813: Network policy protocol support

### PR DESCRIPTION
**Pull request by jab-rh**
- https://issues.redhat.com/browse/OCPBUGS-8813

Version(s): 4.10+

Issue:

Link to docs preview:

[About network policy](https://61107--docspreview.netlify.app/openshift-enterprise/latest/networking/network_policy/about-network-policy.html)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@anuragthehatter, how does this look? Thanks!

**Comment by jab-rh** [comment]
@weliang1, does this look okay? Thanks!

**Comment by weliang1** [comment]
> @weliang1, does this look okay? Thanks!

/lgtm

@asood-rh Could you also help to review this network policy PR? Thanks!

File: modules/nw-networkpolicy-about.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ By default, all pods in a project are accessible from other pods and network end
 
 If a pod is matched by selectors in one or more `NetworkPolicy` objects, then the pod will accept only connections that are allowed by at least one of those `NetworkPolicy` objects. A pod that is not selected by any `NetworkPolicy` objects is fully accessible.
 
+Network policy supports only the TCP, UDP, and SCTP protocols.
```
Comment (snarayan-redhat) [inline modules/nw-networkpolicy-about.adoc:24]:
```suggestion
The network policy supports only the TCP, UDP, and SCTP protocols.
```

**Review by snarayan-redhat — COMMENTED** [review]
Just a minor nit.

**Pull request merged**